src/main/java/reposense/system/LogsManager.java

```diff
@@ -12,7 +12,7 @@
 import java.util.logging.SimpleFormatter;
```

Comment:
Should wrap `Console Handler` and `File Handler` in `@code` in the `addConsoleHandler` and `addFileHandler` functions in this file.